lua/wikis/commons/Class.lua

```diff
@@ -64,14 +64,25 @@ end
 ---@param options ?table
 ---@return T
 function Class.export(class, options)
-	for name, f in pairs(class) do
+	options = options or {}
+
+	local checkFunction = function(functionName)
+		local f = class[functionName]
 		-- We only want to export functions, and only functions which are public (no underscore)
-		if (
-			type(f) == 'function' and
-				(not string.find(name, Class.PRIVATE_FUNCTION_SPECIFIER))
-		) then
-			class[name] = Class._wrapFunction(class[name], options)
+		if type(f) ~= 'function' or string.find(functionName, Class.PRIVATE_FUNCTION_SPECIFIER) then
+			return
+		end
+		class[functionName] = Class._wrapFunction(f, options)
+	end
+
+	if type(options.onlyExport) == 'table' and #options.onlyExport > 0 then
```

Comment:
Maybe `entrypoints` could be an alternative name for `onlyExport`? The latter sounds more like a flag than an array to me.

Reply:
hmm maybe `restrictToEntryPoints`?
because if not specified it still exports all (not prefixed by underscore)